test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py

```diff
@@ -52,11 +52,11 @@ class AccountConnectionWizard:
         "visible": 1,
         "window": names.owncloudWizard_OCC_OwncloudWizard,
     }
-    CHOOSE_LOCAL_SYNC_FOLDER = {
-        "container": names.add_Folder_Sync_Connection_groupBox_QGroupBox,
+    Add_Folder_Sync_Connection_localFolderLineEdit_QLineEdit = {
```

Comment:
```suggestion
   CHOOSE_LOCAL_SYNC_FOLDER = {
```
I think it would be better to use the same name ` CHOOSE_LOCAL_SYNC_FOLDER` as the selector assigned to it is used to choose the local sync directory.